src/app/services/age-verification/age-verification.service.ts

```diff
@@ -10,12 +10,18 @@ export class AgeVerificationService {
 
   performAgeValidation({ age, firstName, lastName }: IUser): string {
     const fullName = `${firstName} ${lastName}`;
-    const userHasLegalAge = age >= this.LEGAL_AGE;
+    const userHasLegalAge = this.isLegalAge(age);
 
     if (userHasLegalAge) {
       return `${fullName} is legal age`;
     }
 
     return `${fullName} is a minor`;
   }
+
+  isLegalAge(age: number): boolean {
```

Comment:
I will suggest you to move this method before `performAgeValidation` since is a good practice to order methods/variables alphabetically.